.github/workflows/ci_on_debian11.yml.eol

```diff
@@ -42,11 +42,9 @@ jobs:
         run: ./ci/install.sh
       - name: test shell
         run: |
-          ./ci/test_shell_espnet1.sh
           ./ci/test_shell_espnet2.sh
           # Note: ESPnet-3 does not have shell, so we will not check.
       - name: test python
         run: |
-          ./ci/test_python_espnet1.sh
```

Comment:
Is it OK to exclude this?
According to https://github.com/espnet/espnet/blob/master/ci/test_python_espnet1.sh#L19, this test seems to include many other espnet2-related tests (maybe, I'm wrong, but I could read like this)

Reply:
I think we can remove this line. The script currently checks:
- flake8 for the espnet directory
- format check with pycodestyle (with several excluded directories)
- tests in test/

For the pycodestyle part, it is true that it checks the espnet2 directory in the current code. However, the exclude settings are exactly the same as those in `test_python_espnet2.sh`, so it will also checked in `test_python_espnet2.sh`.

https://github.com/espnet/espnet/blob/eaf9a83f5f5a7427651b2eb4b6438a9a6427ebf0/ci/test_python_espnet2.sh#L8C1-L16

Reply:
So, all test functions in https://github.com/espnet/espnet/tree/master/test are not related to espnet2?
I think so for some of them, but I'm not sure about all of them.
But maybe we can trust the espnet2 test.
In this case, we can also remove test files in https://github.com/espnet/espnet/tree/master/test (in another PR? Either is fine)